### DISCUSSION TITLE
Bootloader pin

### DISCUSSION THREAD

**Discussion by muriloducatti** [OpenI6X/opentx#341] [General]
Hello everybody!

Unintentionally I ended up taking off the r53 from the board that makes the bootloader and it is not possible to repair it. Is there any direct pin on APM32 where I can trigger bootloader mode?

And congratulations to the developers, the system is excellent!

**Comment by ajjjjjjjj** [comment]
pin name: BOOT0, pin number in LQFP100 package is 94.

**Reply by muriloducatti** [reply]
Muito obrigado!
